src/components/Navigation/components/Navigation.tsx

```diff
@@ -0,0 +1,52 @@
+import Component from "vue-class-component";
+import BaseNavigation from "./BaseNavigation";
+@Component({
+  name: "Navigation",
+})
+class Navigation extends BaseNavigation {
+  render() {
+    return (
+      <div class="flex flex-row h-full">
+        {this.items.map(item => (
+          <div class="relative group overflow-hidden hover:overflow-visible">
+            <a
+              href={item.path}
+              onClick={event => {
+                event.preventDefault();
+                this.triggerItemClick(item);
+              }}
+              class="h-full flex items-center justify-center px-5 py-4 uppercase tracking-widest text-xs"
+            >
+              {item.label}
+            </a>
+            {item.children.length > 0 ? (
+              <div
+                class={`absolute top-100 ${
+                  item.childPlacement === "right" ? "right" : "left"
```

Comment:
Can't you just put the string here? That's kind of like` boolean === true ? true : false`
Wouldn't `${item.childPlacement}-0` be enough? Because the ternary operator only validates whether the string is "right" but not whether it is "left" it could be anything and will get mapped to "left"

Reply:
Good catch, but it is even worse:
I did a string concatenation to build the class and PurgeCSS will remove the classes `left-0` and `right-0`, if they are not used elsewhere.

We have to change the code to this:

```
item.childPlacement === "right" ? "right" : "left" ? "right-0" : "left-0"
```